data/interfaces/default/graphs.html

```diff
@@ -373,14 +373,35 @@ <h4><i class="fa fa-calendar"></i> Total <span class="yaxis-text" style="text-tr
             type: 'get',
             dataType: "json",
             success: function (data) {
-                var select = $('#graph-user');
+                let select = $('#graph-user');
+                let by_id = {};
                 data.sort(function(a, b) {
                     return a.friendly_name.localeCompare(b.friendly_name);
                 });
                 data.forEach(function(item) {
                     select.append('<option value="' + item.user_id + '">' +
                         item.friendly_name + '</option>');
+                    by_id[item.user_id] = item.friendly_name;
                 });
+                select.selectpicker({
+                    countSelectedText: function(sel, total) {
+                        if (sel === 0 || sel === total) {
+                            return 'All users';
+                        } else if (sel > 3) {
```

Comment:
2 usernames can already overflow the container.

```suggestion
                        } else if (sel > 1) {
```